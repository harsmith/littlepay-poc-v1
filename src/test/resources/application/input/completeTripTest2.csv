ID,DateTimeUTC,TapType,StopId,CompanyId,BusID,PAN
1,22-01-2018 13:15:00,ON,Stop1,Company10,Bus37,34343434343434
2,22-01-2018 13:20:00,OFF,Stop3,Company10,Bus37,34343434343434