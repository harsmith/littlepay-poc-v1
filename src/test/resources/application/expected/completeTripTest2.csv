Started,Finished,DurationSecs,FromStopId,ToStopId,ChargeAmount,companyId,BusID,PAN,Status
22-01-2018 13:15:00,22-01-2018 13:20:00,300,Stop1,Stop3,$7.30,Company10,Bus37,34343434343434,COMPLETED
